File: 09-09/adts/Pair.java
package adts;

public class Pair<T1, T2> {
  public T1 fst;
  public T2 snd;

  public Pair() {
  }

  public Pair(T1 t1, T2 t2) {
    fst = t1;
    snd = t2;
  }
}

File: 09-09/adts/pair.cpp
#pragma once

template <class T1, class T2>
struct pair
{
  T1 fst;
  T2 snd;

  pair() {}
  pair(T1 t1, T2 t2)
  {
    fst = t1;
    snd = t2;
  }
};
